PM2023-001 - Total node failure.md

```diff
@@ -0,0 +1,63 @@
+# Total node failure
+
+## Summary
+
+During an unscheduled node rollover the Kubernetes cluster was suddenly left without any worker nodes. Multiple node rollover attempts later, the cluster was back in a working state with the expected amount of worker nodes.
+
```

Comment:
Maybe start by mention that logs weren't being shipped which was the trigger for the incident.

Reply:
Updated the summary in https://github.com/dfds/postmortems/pull/27/commits/bc5d728c7828d8353a58160e925ba6cde1837ff1. Any better?

Reply:
Much better, thank you :)